Sources_v3/APIClient/Endpoints/Payloads/DevicePayloads.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+struct DevicePayload: Decodable, Equatable {
+    private enum CodingKeys: String, CodingKey {
+        case id
+        case createdAt = "created_at"
+    }
+    
+    /// Device identifier.
+    let id: String
+    /// Date the device was created for the user.
+    let createdAt: Date?
+    
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        id = try container.decode(String.self, forKey: .id)
+        createdAt = try container.decodeIfPresent(Date.self, forKey: .createdAt)
+    }
+    
+    /// Initializer only used in dummy data/tests.
+    init(id: String, createdAt: Date? = .init()) {
```

Comment:
nit: 
If it is used in tests only, maybe it makes sense to move it there? 
Or even use `init(id: String, createdAt: Date?)` directly?

Reply:
Comment was incorrect, this init is also used when a device is added (see `CurrentUserController.addDevice`)
Fixed in 46f926451006e1b0be9125afc4109583cef1a9f7